api/oc_network_events_internal.h

```diff
@@ -78,6 +78,14 @@ typedef struct oc_network_interface_cb
 void oc_network_interface_event(oc_interface_event_t event);
 #endif /* OC_NETWORK_MONITOR */
 
+/**
+ * @brief Returns the network event queue length for the device
+ * 
+ * @param device valid device index
+ * @return number of events in the queue
+ */
+int oc_get_network_events_queue_length(size_t device);
```

Comment:
return `size_t` instead of `int`

Reply:
`oc_get_network_events_queue_length` -> `oc_network_get_events_queue_length`, so we keep the `oc_network` 'namespace' prefix

Reply:
Done, new return type changed to `size_`t and the method was renamed to
`size_t oc_network_get_event_queue_length(size_t device)`